### PR TITLE
Remove offensive words

### LINKED ISSUE #3: You might wanna run a censorship pass on the assets.
image
https://github.com/skedwards88/palette/blob/c7666a6cd1db7f06102fcbbec93fab9ce8b500ea/src/common/wordLists/compiled/uncommonWords.json#L10152
I don't know how generation of puzzles works, but I'm suspecting that after a candidate is generated — it's validated against the assets again to make sure there's not accidentally-emerged extra words present, which would be ambiguous and give players differing results of a daily puzzle.
I suggest you move uncooth words to a blacklist instead of getting rid of them. And use that blacklist in addition to the main list when validating.

### PR DESCRIPTION
**Pull request by skedwards88**
Closes https://github.com/skedwards88/palette/issues/3
Follow up to https://github.com/skedwards88/word_games/pull/14

Removes potentially offensive words from the game. Words that are semi-offensive were moved from the "common" list to the "uncommon" list.

### CODE REVIEW

File: src/common/wordLists/compiled/commonWords.json [hunk 1]
```diff
@@ -131,7 +131,6 @@
   "APPLES",
   "APRONS",
   "ARCADE",
-  "ARCANA",
```
Comment (skedwards88) [inline src/common/wordLists/compiled/commonWords.json:134]: un-related to the offensiveness, this was just deemed to not be "common"

File: src/common/wordLists/compiled/commonWords.json [hunk 2]
```diff
@@ -673,7 +672,6 @@
   "CORDON",
   "CORNEA",
   "CORNER",
-  "CORNET",
```
Comment (skedwards88) [inline src/common/wordLists/compiled/commonWords.json:676]: un-related to the offensiveness, this was just deemed to not be "common"

File: src/common/wordLists/compiled/commonWords.json [hunk 3]
```diff
@@ -2606,7 +2601,6 @@
   "SEABED",
   "SEALED",
   "SEAMAN",
-  "SEAMEN",
```
Comment (skedwards88) [inline src/common/wordLists/compiled/commonWords.json:2609]: un-related to the offensiveness, this was just deemed to not be "common"

Comment (Erquint) [reply src/common/wordLists/compiled/commonWords.json:2609]: > un-related to the offensiveness

😅*Giving my chuckling inner 5yo a stern introspective look.*

**Pull request merged**